datahub-web-react/src/app/search/sidebar/BrowseSidebar.tsx

```diff
@@ -37,29 +44,80 @@ const SidebarBody = styled.div<{ visible: boolean }>`
 
 type Props = {
     visible: boolean;
-    width: number;
 };
 
-const BrowseSidebar = ({ visible, width }: Props) => {
+const EntityAggregationsDraggableList = ({entityAggregations}: any) =>{
+    const [updateEntityAggregations, setupdateEntityAggregations] = useState(entityAggregations)
+
+    useEffect(()=>{
+        setupdateEntityAggregations(entityAggregations)
+    },[entityAggregations]);
+
+    function handleOnDragEnd(result) {
+        if (!result.destination) return;
+    
+        const items = Array.from(updateEntityAggregations);
+        const [reorderedItem] = items.splice(result.source.index, 1);
+        items.splice(result.destination.index, 0, reorderedItem);
+    
+        setupdateEntityAggregations(items);
+      }
+    return (
+        <DragDropContext onDragEnd={handleOnDragEnd}>
+            <Droppable droppableId="entity">
+                {(provided) => (
+                <div className="entity" {...provided.droppableProps} ref={provided.innerRef}>
+                    {updateEntityAggregations?.map((entityAggregation, index)=> {
+                    return (
+                        <Draggable key={entityAggregation.value} draggableId={entityAggregation.value} index={index}>
```

Comment:
Why do we need all of the draggable stuff here? 

Can we just follow the same pattern we are using on the other sidebar? This will require new developers get up to speed on this draggable thing

I might be missing some advantage to introducing this, though. If yes, can you please explain?

Reply:
I was confused between draggable and resizer as ticket having title  "allow the sidebar size to be draggable" after I post video on linear [@Aseem Bansal](https://acryldata.slack.com/team/U02NWCS4VB8) said  we need Resizer. you can also check conversation on ticket

Reply:
I have remove Draggable